tests/test_models.py

```diff
@@ -127,7 +145,25 @@ def test_job_output_creation():
         job_output = JobOutput(**output)
 
 def test_job_creation():
-    job_record = db_utils.read_json("job_record.json")
+    job_record = db_utils.read_json("unsubmitted_job_record.json")
     job = Job(**job_record)
     assert job.id == job_record["id"]
-    assert isinstance(job.workflow, JobWorkflow)
\ No newline at end of file
+    assert isinstance(job.workflow, JobWorkflow)
+
+
+def test_get_base_workflow_execution_keys():
```

Comment:
Why doesn't this match what is in the schema class? Name, description, etc are missing.

Reply:
`WorkflowExecution.__annotations__.items()` doesn't return keys from it's parent classes.  In any case the function wasn't actually doing anything useful so it's gone